data/editor.config

```diff
@@ -444,6 +444,31 @@
         <include group="poi" />
         <include field="internet" />
       </type>
+      <type id="healthcare-alternative" group="health">
+        <include group="poi" />
```

Comment:
It would make sense to create a group poi_internet and use it here too. A private doctor can have wifi for his patients.

Reply:
I agree.
Changed https://github.com/organicmaps/organicmaps/pull/4053/commits/7dd0719b3f287dbad1ade77d2d1bbb0f61504ec0